server/routes/api/place.py

```diff
@@ -432,14 +523,18 @@ def get_ranking_url(containing_dcid,
     return url
 
 
-@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
```

Comment:
i guess ideally we could use @cache.cached for all direct flask endpoints?

Reply:
Hm, yes, especially now that there is an implicit get parameter that is used across endpoints.